rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinUtilsTest.java

```diff
@@ -1152,6 +1143,57 @@ void addSpendingFederationBaseScript_forSegwitFed_shouldAddExpectedInputWitness(
         assertEquals(oneChunkSegwitScriptSig, transaction.getInput(inputIndex).getScriptSig());
     }
 
+    @Test
+    void buildSegwitScriptSig_buildsScriptSigWithExpectedStructure() {
```

Comment:
Missing unit tests for NonSegwitScriptSig, Empty, null.

Reply:
Updated.